ContractsApi.Tests/Dockerfile

```diff
@@ -19,10 +19,10 @@ RUN apt-get update && apt-get install -y openjdk-17-jdk
 #  && apk upgrade --no-cache \
 #  && apk add --no-cache openjdk-17-jdk
 
-RUN dotnet tool install --global dotnet-sonarscanner
-ENV PATH="$PATH:/root/.dotnet/tools"
+ RUN dotnet tool install --global dotnet-sonarscanner
+ ENV PATH="$PATH:/root/.dotnet/tools"
```

Comment:
The indentation of all the commands in this file should be equal.

Reply:
resolved, ty for the heads up